cocos/core/pipeline/custom/index.jsb.ts

```diff
@@ -23,11 +23,6 @@
  THE SOFTWARE.
  */
 
-/**
- * @packageDocumentation
- * @module custom-pipeline
- */
-
```

Comment:
Documentation is accidentally removed.

Reply:
This is unnecessary now, we don't rely on this for API module any more